src/odemis/gui/cont/tabs.py

```diff
@@ -988,6 +999,13 @@ def _create_views(self, main_data, viewports):
                     v["fov_hw"] = main_data.ebeam
                     vp.canvas.fit_view_to_next_image = False
 
+        if main_data.fib_scanner:
+            vpv[viewports[1]]['stream_classes'] = FIBStream
+            # Remove stage control since the pixel size/magnification is unknown for the FIB.
+            vpv[viewports[1]].pop('stage')
+            if CAN_DRAG in viewports[1].canvas.abilities:
+                viewports[1].canvas.disable_drag()
```

Comment:
Let's not worry about this. Firstly, because moving the stage has actually effect on the FIB view, so it could still be useful (just not in the case of the ENZEL). Most importantly, that's just for internal testing, so let's not worry about it, and keep the code as simple as possible.